docs/source/index.rst

```diff
@@ -16,13 +16,13 @@ Unified Model Serving Framework
 
 BentoML let you create machine learning powered prediction service in minutes and bridges the gap between data science and DevOps.
 
+👉 `Pop into our Slack community! <https://join.slack.bentoml.org>`_ We're happy to help with any issue you face or even just to meet you and hear what you're working on :)
```

Comment:
could we also make this change to the README file https://github.com/bentoml/BentoML/blob/main/README.md and the docker hub readme template file here: https://github.com/bentoml/BentoML/blob/main/docker/templates/docs/README.md.j2 - this docker template file will be rendered on the docker hub page here:  https://hub.docker.com/r/bentoml/bento-server

Reply:
ah, nice, I didn't know that. cool yea i'll do it

Reply:
done